test/ng/sceSpecs.js

```diff
@@ -464,6 +464,38 @@ describe('SCE', function() {
             '$sce', 'insecurl', 'Blocked loading resource from url not allowed by $sceDelegate policy.  URL: foo');
         }
       ));
+
+      describe('when the document base URL has changed', function() {
+        var baseElem = window.document.createElement('BASE');
+        var cfg = {whitelist: ['self'], blacklist: []};
+        baseElem.setAttribute('href', window.location.protocol + '//foo.example.com/path/');
+        beforeAll(function() {
```

Comment:
If there is no particular reason, I would prefer re-creating/adding/removing the `<base>` element for **each** test.